queries.sql

```diff
@@ -7,4 +7,26 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.5 AND 17.3;
+UPDATE animals SET species = 'unspecified';
+BEGIN ;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokimon' WHERE species IS NULL;
+COMMIT ;
```

Comment:
- [ ] Good job update the animals table, 👍 however you should verify that the change has been made before the commit and after the commit 👍 ,Kindly check that 👀 

```sql
BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```